growbox/systime_settings.py

```diff
@@ -1,5 +1,9 @@
-import os
+import subprocess
 
 
 def set_systime(date_time):
-    os.system("sudo date --set='" + date_time + "'")
+    command = ["sudo", "date", "--set=" + date_time]
+    try:
+        subprocess.run(command, check=True)
+    except subprocess.CalledProcessError:
+        raise
```

Comment:
If you always re-raise, why catch the exception then?

Reply:
Fix it.